src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
+    public DifferentColors getRandomColor() {
+        int index = new Random().nextInt(DifferentColors.values().length);
```

Comment:
```suggestion
public class ColorSupplier extends Ball {
    private Random random = new Random();
    
    public DifferentColors getRandomColor() {
        int index = random.nextInt(DifferentColors.values().length);
```
вивести на рівень класу означає зробити так щоб об'єкт був доступний для різних методі в рамках одного класу замість одного методу